x/auth/tx/builder.go

```diff
@@ -287,11 +287,17 @@ func (w *wrapper) SetSignatures(signatures ...signing.SignatureV2) error {
 	rawSigs := make([][]byte, n)
 
 	for i, sig := range signatures {
-		var modeInfo *tx.ModeInfo
+		var (
```

Comment:
[question] should this be upstreamed [here](https://github.com/cosmos/cosmos-sdk/blob/7799bba7bc889288607576aa11655b5fc31a2da9/x/auth/tx/builder.go#L284-L307)? If not, should we add it to the [README modifications](https://github.com/celestiaorg/cosmos-sdk?tab=readme-ov-file#modifications).

[question] do we want a unit test so that this doesn't get reverted when we pull from upstream?

^ sorry I repeat this a lot. I'm paranoid about accidentally reverting stuff when we pull from upstream because it's _really_ difficult to catch a regression on those PRs.

Reply:
Yeah was planning on adding a unit test and I also plan to upstream it. 

I had opened an issue on the SDK and was waiting for their response

Reply:
For posterity, the Cosmos SDK issue and PR:
- https://github.com/cosmos/cosmos-sdk/issues/19094
- https://github.com/cosmos/cosmos-sdk/pull/19106